internal/pkg/handlers/handlers.go

```diff
@@ -0,0 +1,266 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/serjyuriev/shortener/internal/pkg/shorty"
+	"github.com/serjyuriev/shortener/internal/pkg/storage"
+)
+
+type ContextKey string
+
+var contextKeyUID = ContextKey("uid")
```

Comment:
А для чего это так?

Reply:
Если своим типом не переопределять, то staticcheck ругается: `should not use built-in type string as key for value; define your own type to avoid collisions (SA1029)`, поэтому так сделал.